app/mm-redux/selectors/entities/apps.ts

```diff
@@ -19,6 +19,22 @@ export function getAppsBindings(state: GlobalState, location?: string): AppBindi
     return state.entities.apps.bindings;
 }
 
+export function getThreadAppsBindingsChannelID(state: GlobalState): string {
+    return state.entities.apps.threadBindingsChannelID;
+}
+
+export function getThreadAppsBindings(state: GlobalState, location?: string): AppBinding[] {
+    if (!state.entities.apps.threadBindings) {
+        return [];
+    }
```

Comment:
Should this function be memoized?

Reply:
I removed these functions and left the "create selector" ones.